packages/ui/src/components/va-navbar/VaNavbar.demo.vue

```diff
@@ -19,7 +19,35 @@
       </va-navbar>
     </VbCard>
 
-    <VbCard style="width: 100%;" title="Custom color">
+    <VbCard title="Slots" style="width: 100%;">
+      <va-navbar color="primary">
+        <template #left>
```

Comment:
Would be nice to have a demo with combination of different slots (rl, rc, lc, rcl)

Reply:
Only for the demo or  for the docs too?